### ISSUE TITLE
Placeholder %jobsr_user_points% is not working with PAPI

### ISSUE THREAD

**Issue by Deathinflames** [Zrips/Jobs#457]
***Description of issue:**
The placeholder %jobsr_user_points% doesn't appear to be working properly. 
When using "/jobs placeholders -p:2", the result is showing the proper amount however the PAPI placeholder has no output and only returns the placeholder name again when using the command "/papi parse me %jobsr_user_points%"

**Server Type (Spigot/Paperspigot/etc):** 
Paperspigot

**Server Version (using `/ver`):** 
1.13.2

**Relevant plugins (Delete if this isn't needed):**
Jobs Reborn 4.12.0
PlaceholderAPI 2.10.2

**Comment by montlikadani** [comment]
You should report this to PAPI, because the PAPI does not updated the Jobs to latest version to get new placeholders.

**Comment by MrTrinax** [comment]
Sorry for bumping this issue, but didn't want to open another.
Does Jobs Reborn support PlaceholderAPI and is it possible to get placeholders onto scoreboard without PlaceholderAPI?

**Comment by montlikadani** [comment]
@MrTrinax Jobs reborn is only registering custom placeholders to PAPI and can be get from PAPI. Without PAPI for scoreboard plugins are not work the placeholders.

**Comment by MrTrinax** [comment]
I'm sorry, but could you rephrase that?
